frontends/qt/resources/MacOS/Info.plist

```diff
@@ -40,5 +40,8 @@
 
 	<key>LSApplicationCategoryType</key>
 	<string>public.app-category.finance</string>
+
+	<key>NSCameraUsageDescription</key>
```

Comment:
Does this just show the message, or does it actually enable the use of the camera (e.g. doesn't work if missing)?

Would be great to add this info to the commit message or as a comment here if possible.

Reply:
```
The app’s Info.plist must contain an NSCameraUsageDescription key with a string value explaining to the user how the app uses this data.
```
https://useyourloaf.com/blog/privacy-settings-in-ios-10/

Reply:
Yes, this solved the issue I had quite reliably. I amended the commit message. ✅